File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.js
function solution(n, control) {
    return control.split('').reduce((answer, sumCount) => {
        switch (sumCount) {
            case 'w':
                return answer + 1;
            case 's':
                return answer - 1;
            case 'd':
                return answer + 10;
            case 'a':
                return answer - 10;
            default:
                return answer;
        }
    }, n);
}

File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.py
def solution(n, control):
    answer = n
    for cha in control:
        if cha == "w":
            answer += 1
        elif cha == "s":
            answer -= 1
        elif cha == "d":
            answer += 10
        else:
            answer -= 10
    return answer